plugins/ssl/cert_validity

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+#
+# check the certificate validity of your certfificates
```

Comment:
Please use a perldoc header here.

The [ps_ plugin](https://github.com/munin-monitoring/munin/blob/master/plugins/node.d/ps_) could be a good example.

Reply:
changed to perldoc